CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.2
+
+### Calc Functions Interpolation Migrator
+
+* Fix operators regex to remove unnecessary parenthesis in SCSS variables.
```

Comment:
This describes an implementation detail rather than the actual user-visible fix. This should instead say something about fixing it so it no longer interprets a dash in a variable name as a minus sign.

Reply:
Thanks!